project/Dependencies.scala

```diff
@@ -8,16 +8,23 @@
  */
 
 import sbt._
+import Keys._
 
 object Dependencies {
   // Keep in sync with .github CI build
   val Scala212 = "2.12.18"
   val Scala213 = "2.13.12"
-  val ScalaVersions = Seq(Scala212, Scala213)
+  val Scala3 = "3.3.3"
+  val ScalaVersions = Seq(Scala212, Scala213, Scala3)
 
   val PekkoVersion = PekkoCoreDependency.version
+  val PekkoBinaryVersion = PekkoVersion.take(3)
 
-  val SlickVersion = "3.3.3"
+  val LogbackForSlf4j1Version = "1.2.13"
+  val LogbackForSlf4j2Version = "1.3.14"
+  val LogbackVersion = if (PekkoBinaryVersion == "1.0") LogbackForSlf4j1Version else LogbackForSlf4j2Version
```

Comment:
in pekko-persistence-jdbc 1.1, I think we should just use the LogbackForSlf4j2Version regardless of Pekko version

Slick 3.5.0 is slf4j based anyway

Code built with slif4j v1 works with slf4j v2 at runtime but this is not necessarily true the other way around.

Reply:
Sure, I just copied this from pekko-connectors